[2015-12-06 12:31:46] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7016
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9573): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2979): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9573): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9563): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:31:46] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7016
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:40:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:40:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:40:50] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:40:50] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:43:15] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:43:15] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:43:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:43:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:43:37] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:43:37] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:44:49] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:44:49] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:46:39] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:46:39] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:48:41] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:48:41] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:49:38] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:49:38] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-29 07:29:47] local.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in C:\xampp\htdocs\oslonews\app\Category.php:52
Stack trace:
#0 C:\xampp\htdocs\oslonews\app\Category.php(52): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\oslonews\app\Providers\MenuServiceProvider.php(47): App\Category::getTopMenuHubCategories('international-e...')
#2 [internal function]: App\Providers\MenuServiceProvider->App\Providers\{closure}(Object(Illuminate\View\View))
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9749): call_user_func_array(Object(Closure), Array)
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14100): Illuminate\Events\Dispatcher->fire('composing: topm...', Array)
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14482): Illuminate\View\Factory->callComposer(Object(Illuminate\View\View))
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#7 C:\xampp\htdocs\oslonews\storage\framework\views\0b9c86839e8965eda89e9ef2e6e2fd10(159): Illuminate\View\View->render()
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#13 C:\xampp\htdocs\oslonews\storage\framework\views\9698c9b52c283cddfbf00a1591f743da(43): Illuminate\View\View->render()
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#17 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#18 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#23 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#29 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#41 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#44 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#50 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#51 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#52 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#54 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#55 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#56 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#57 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#58 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#59 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#60 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#61 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#62 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#63 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#64 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#65 {main}

Next exception 'ErrorException' with message 'Trying to get property of non-object (View: C:\xampp\htdocs\oslonews\resources\views\user.blade.php)' in C:\xampp\htdocs\oslonews\app\Category.php:52
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14628): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\oslonews\storage\framework\views\9698c9b52c283cddfbf00a1591f743da(43): Illuminate\View\View->render()
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#46 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#52 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#55 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#56 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#57 {main}

Next exception 'ErrorException' with message 'Trying to get property of non-object (View: C:\xampp\htdocs\oslonews\resources\views\user.blade.php) (View: C:\xampp\htdocs\oslonews\resources\views\user.blade.php)' in C:\xampp\htdocs\oslonews\app\Category.php:52
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14628): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#46 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#51 {main}  
[2016-01-08 04:26:53] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1049] Unknown database 'oslotimeshub'' in C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:55
Stack trace:
#0 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(55): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(22): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11419): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11415): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11330): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11285): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11162): Illuminate\Database\DatabaseManager->connection(NULL)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11149): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(10596): Illuminate\Database\Eloquent\Model->getConnection()
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(10573): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(10563): Illuminate\Database\Eloquent\Model->newQueryWithoutScopes()
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11246): Illuminate\Database\Eloquent\Model->newQuery()
#12 [internal function]: Illuminate\Database\Eloquent\Model->__call('get', Array)
#13 [internal function]: App\Hub->get()
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11252): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\oslonews\app\Hub.php(25): Illuminate\Database\Eloquent\Model::__callStatic('get', Array)
#16 C:\xampp\htdocs\oslonews\app\Hub.php(25): App\Hub::get()
#17 C:\xampp\htdocs\oslonews\app\Providers\MenuServiceProvider.php(44): App\Hub::getHubs()
#18 [internal function]: App\Providers\MenuServiceProvider->App\Providers\{closure}(Object(Illuminate\View\View))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9749): call_user_func_array(Object(Closure), Array)
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14100): Illuminate\Events\Dispatcher->fire('composing: topm...', Array)
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14482): Illuminate\View\Factory->callComposer(Object(Illuminate\View\View))
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#23 C:\xampp\htdocs\oslonews\storage\framework\views\56a00fdf9a281321197aae21f75d6705(157): Illuminate\View\View->render()
#24 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#26 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#29 C:\xampp\htdocs\oslonews\storage\framework\views\9698c9b52c283cddfbf00a1591f743da(43): Illuminate\View\View->render()
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#32 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#35 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#38 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#39 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#44 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#46 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#48 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#51 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#52 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#55 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#56 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#57 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#58 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#60 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#61 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#63 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#64 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#65 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#66 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#67 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#68 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#69 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#70 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#71 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#72 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#73 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#74 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#75 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#76 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#77 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#78 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#79 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#80 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#81 {main}

Next exception 'ErrorException' with message 'SQLSTATE[HY000] [1049] Unknown database 'oslotimeshub' (View: C:\xampp\htdocs\oslonews\resources\views\default.blade.php)' in C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:55
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14628): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(PDOException), 1)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\oslonews\storage\framework\views\9698c9b52c283cddfbf00a1591f743da(43): Illuminate\View\View->render()
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#46 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#52 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#55 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#56 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#57 {main}

Next exception 'ErrorException' with message 'SQLSTATE[HY000] [1049] Unknown database 'oslotimeshub' (View: C:\xampp\htdocs\oslonews\resources\views\default.blade.php) (View: C:\xampp\htdocs\oslonews\resources\views\default.blade.php)' in C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:55
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14628): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#46 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#51 {main}  
[2016-01-08 04:27:00] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1049] Unknown database 'oslotimeshub'' in C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:55
Stack trace:
#0 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(55): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(22): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11419): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11415): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11330): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11285): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11162): Illuminate\Database\DatabaseManager->connection(NULL)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11149): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(10596): Illuminate\Database\Eloquent\Model->getConnection()
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(10573): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(10563): Illuminate\Database\Eloquent\Model->newQueryWithoutScopes()
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11246): Illuminate\Database\Eloquent\Model->newQuery()
#12 [internal function]: Illuminate\Database\Eloquent\Model->__call('where', Array)
#13 [internal function]: App\Category->where('parent_id', '=', '0')
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11252): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\oslonews\app\Category.php(80): Illuminate\Database\Eloquent\Model::__callStatic('where', Array)
#16 C:\xampp\htdocs\oslonews\app\Category.php(80): App\Category::where('parent_id', '=', '0')
#17 C:\xampp\htdocs\oslonews\app\Http\Controllers\HubsController.php(41): App\Category::getCategories()
#18 [internal function]: App\Http\Controllers\HubsController->create()
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8866): call_user_func_array(Array, Array)
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8935): Illuminate\Routing\Controller->callAction('create', Array)
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\HubsController), Object(Illuminate\Routing\Route), 'create')
#22 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\HubsController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'create')
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'create')
#29 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#39 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#41 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#44 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#50 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#51 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#52 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#54 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#55 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#56 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#57 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#58 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#59 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#60 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#61 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#62 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#63 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#64 {main}  
[2016-01-08 04:58:02] local.ERROR: exception 'ErrorException' with message 'chdir(): No such file or directory (errno 2)' in C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:36
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'chdir(): No suc...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php(36): chdir('C:\\xampp\\htdocs...')
#2 [internal function]: Illuminate\Foundation\Console\ServeCommand->fire()
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1182): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Console\Command.php(150): Illuminate\Container\Container->call(Array)
#5 C:\xampp\htdocs\oslonews\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\xampp\htdocs\oslonews\vendor\symfony\console\Application.php(838): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\xampp\htdocs\oslonews\vendor\symfony\console\Application.php(189): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Foundation\Console\ServeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\xampp\htdocs\oslonews\vendor\symfony\console\Application.php(120): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\xampp\htdocs\oslonews\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2016-01-08 05:02:45] local.ERROR: exception 'ErrorException' with message 'chdir(): No such file or directory (errno 2)' in C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:36
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'chdir(): No suc...', 'C:\\xampp\\htdocs...', 36, Array)
#1 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php(36): chdir('C:\\xampp\\htdocs...')
#2 [internal function]: Illuminate\Foundation\Console\ServeCommand->fire()
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1182): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Console\Command.php(150): Illuminate\Container\Container->call(Array)
#5 C:\xampp\htdocs\oslonews\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\xampp\htdocs\oslonews\vendor\symfony\console\Application.php(838): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\xampp\htdocs\oslonews\vendor\symfony\console\Application.php(189): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Foundation\Console\ServeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\xampp\htdocs\oslonews\vendor\symfony\console\Application.php(120): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\xampp\htdocs\oslonews\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\xampp\htdocs\oslonews\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2016-01-12 09:00:12] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Intervention\Image\ImageServiceProvider' not found' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:6916
Stack trace:
#0 {main}  
